oak_runtime/src/lib.rs

```diff
@@ -629,6 +637,20 @@ impl Runtime {
         node_privilege.downgrade_label(initial_label)
     }
 
+    /// Returns the effective label for `initial_label` in the context of the Node, taking into
+    /// acount whether the downgrade privilege should be applied.
+    fn get_node_effective_label(
```

Comment:
Nice! Though IIUC the label is not always a node label, right?

Reply:
The "node" part was to indicate that it is in the context of the node's privilege (when used), but agree that it is confusing. Renamed.